PushButton/Node/PushButton/server.js

```diff
@@ -0,0 +1,71 @@
+var uwp = require('uwp');
+uwp.projectNamespace('Windows');
+
+var BUTTON_PIN = 5;
+var LED_PIN = 6;
+
+var ledPin;
+var buttonPin;
+var ledPinValue = Windows.Devices.Gpio.GpioPinValue.high;
+
+(function () { 
+    initGPIO();
+})();
+
+function initGPIO(){
+    var gpio = Windows.Devices.Gpio.GpioController.getDefault();
+    
+    // Log an error if there is no GPIO controller
+    if (gpio == null) {
+        console.log('There is no GPIO controller on this device.');
+        return;
+    }
+    
+    var buttonPinResult = gpio.tryOpenPin(BUTTON_PIN, Windows.Devices.Gpio.GpioSharingMode.exclusive);
+    var ledPinResult = gpio.tryOpenPin(LED_PIN, Windows.Devices.Gpio.GpioSharingMode.exclusive);
+    
+    // Log an error if the GPIO pins can not be opened
+    // GpioOpenStatus enumeration - https://msdn.microsoft.com/en-us/library/windows/apps/windows.devices.gpio.gpioopenstatus.aspx
+    if (!buttonPinResult.succeeded) {
```

Comment:
nit: just a styling preference to maybe do it this way

var buttonPinResult = ...
if((!buttonPinResult.succeeded)...

var letPinResult  = ...
if (!ledPinResult.succeeded) ...